### DISCUSSION TITLE
Mobile Theme

### DISCUSSION THREAD

**Discussion by aashish108** [TriliumNext/Trilium#3575] [Q&A]
Hey all. I have themed the desktop Trilium but does anyone know if its possible to style the mobile frontend also?

**Comment by zadam** [comment]
Hi, it wasn't possible in the past, but now the same theme is applied also to the mobile frontend, e.g.:

<img width="491" alt="image" src="https://user-images.githubusercontent.com/617641/215898844-d24b3ebd-9e37-4513-bf8c-07e5e867d613.png">
